server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/roles.scss

```diff
@@ -167,6 +167,13 @@ $role-icon-hover-color: #000;
   }
 }
 
+.gocd-role-icon {
+  background:          image-url("cruise.ico") no-repeat center;
```

Comment:
Can we use a high resolution png instead of the 16px icon?

https://github.com/gocd/plugin-api.go.cd/blob/master/source/layouts/layout.erb#L24.